docs/dask.md

```diff
@@ -1,54 +1,81 @@
+"mintpy.networkInversion.numWorkers = all" will allocate `multiprocessing.cpu_count()` number of workers to the dask computation.
 
-**walltime** - this is the maximum amount of your time your job is permitted to run on a compute node before it is killed by the node. Walltimes are required so as to ensure users and jobs cannot hog nodes for extended periods of time without doing any substantial computations. Most queues have maximum allowed compute times, after which still running jobs will be terminated, but this is often on the order of 48+ hours. It is recommended to use relatively short walltimes for `MintPy` (the test dataset uses `wall time: 00:30:00` (30 minutes)), but not too short, or your job may never connect to the dask scheduler and finish its computation (when in doubt, start small and work towards larger wall times as needed). Note that the walltime format changes slightly depending on the job scheduler that is in place on your HPC system. For `LSF` schedulers, walltime is accepted and expected to be in `HH:MM` format, while `SLURM` and `PBS` schedulers only accept walltime in `HH:MM:SS` format.  
+### Test on Stampede2 ###
 
-**queue** - this the scheduler queue to submit your jobs to. Most systems have several different queues with different amounts and types of resources you can request. For this reason, it is important to check with your HPC cluster admin or documentation to determine the most appropriate queue to `MintPy` on and what resources are available on that queue. Requesting for more resources (core, memory, nodes, etc.) on a queue than are available often leads to your jobs being rejected from queue, and can, on some systems, lead to fines or account suspensions.
+To show the run time improvement, we test three datasets (Galapagos, Fernandina, and Kilauea) with different number of cores on a compute node in the [Stampede2 cluster's skx-normal queue](https://portal.tacc.utexas.edu/user-guides/stampede2#overview-skxcomputenodes). Results are as below:
+ 
+| Property              | Fernandina Dataset | Galapagos Dataset | Kilauea Dataset            |
+|-----------------------|--------------------|-------------------|----------------------------|
+| Input File Size       | 0.623 GB           | 0.233 GB          | 15.0 GB                    |
+| Dask Configuration    | None               | None              | None                       |
+| Cluster Memory Used   | 16 GB              | 16 GB             | 16 GB (64 GB for 1-2 cores)|
+| Chunk Size Used       | 100e6 (1 GB)       | 100e6 (1 GB)      | 100e6 (1 GB)               | 
+ 
+![dask-local-cluster-performance](resources/dask-local-cluster-performance.png "Dask LocalCluster Peformance")
```

Comment:
Could you move this figure file to `MintPy-tutorial/docs` and make a link to here? This will keep the MintPy code repo small in size.

Here is one example:
```
<p align="center">
  <img src="https://yunjunzhang.files.wordpress.com/2019/02/fernandinasendt128_ge-1.png">
</p>
```

Reply:
I couldn’t find a way to reference the image if it’s placed in a separate repository, since the files in a repo don’t have a publicly accessible file name that can be used as a link.

I’ll let you figure this out, as I don’t think it’s a big deal.

Reply:
I could upload this figure to my personal website if you don't mind. Let me know please @Ovec8hkin .

Reply:
I agree it's no big idea, just to keep the codebase size small.